server/checks/PullRequestTasks.js

```diff
@@ -0,0 +1,82 @@
+import Check, { getPayloadFn } from './Check'
+import { PULL_REQUEST } from '../model/GithubEvents'
+import { logger } from '../../common/debug'
+
+const CHECK_TYPE = 'pullrequesttasks'
+const CONTEXT = 'zappr/pr/tasks'
+const info = logger(CHECK_TYPE, 'info')
+const error = logger(CHECK_TYPE, 'error')
+const debug = logger(CHECK_TYPE)
+const createStatePayload = getPayloadFn(CONTEXT)
+
+export function hasOpenTasks(pr) {
+  const regex = /(-|\*) \[(\s|x)\]/g;
+  let count = 0;
+  let repoName = pr.head.repo.full_name;
+  let prNo = pr.number;
+  let match;
+
+  while ((match = regex.exec(pr.body)) !== null) {
+    // This is necessary to avoid infinite loops with zero-width matches
+    if (match.index === regex.lastIndex) {
+      regex.lastIndex++;
+    }
+
+    let task = match[0];
+    let marker = match[2];
+
+    debug(`${repoName}#${prNo}: Processing task: ${task}`);
+
+    if (marker === " ") {
+      count++;
+    }
+  }
+
+  info(`${repoName}#${prNo}: PR has ${count} open tasks.`);
+
+  return count > 0;
+}
+
+export default class PullRequestTasks extends Check {
+  static TYPE = CHECK_TYPE;
+  static CONTEXT = CONTEXT;
+  static HOOK_EVENTS = [PULL_REQUEST];
+  static NAME = 'Pull request tasks check'
+
+  constructor(github) {
+    super()
+    this.github = github;
+  }
+
+  async execute(config, hookPayload, token) {
+    const {action, repository, number, pull_request} = hookPayload
+    const repoOwner = repository.owner.login
+    const repoName = repository.name
+    const fullName = repository.full_name
+    let status;
+
+    try {
+      // TODO: configuration settings
```

Comment:
In fact I couldn't think of any settings at all for this feature. It's only about enabling/disabling it at this moment. But I placed a TODO just in case someone wants to add some configuration flags in the future.